### PR TITLE
Update enabling cors doc for choreo connect

### PR DESCRIPTION
**Pull request by VajiraPrabuddhaka**
## Purpose

- Remove unused docs for "Enabling CORS"
- Modified the doc for "Enabling CORS"

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/enabling-cors.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+#### Enable CORS configuration for API resources (API level)
 
-You can add CrossOrigin Resource Sharing ( **CORS** ) configurations for each API (at API level) using the OpenAPI extension **x-wso2-cors.** The following code snippet depicts the usage of the `         x-wso2-cors        ` extension. For more information, see the [detailed sample OpenAPI definition with CORS level configuration](https://github.com/wso2/product-microgateway/blob/master/samples/cors_sample.yaml) .
+If you are following the dev-first approach ([Deploy API via CLI tool](https://apim.docs.wso2.com/en/4.0.0/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect/#via-the-cli-tool-apictl))You can add CrossOrigin Resource Sharing ( **CORS** ) configurations for each API (at API level) using the OpenAPI vendor extension **x-wso2-cors** in the API definition. The following code snippet depicts the usage of the `         x-wso2-cors        ` extension. For more information, see the [detailed sample OpenAPI definition with CORS level configuration](https://github.com/wso2/product-microgateway/blob/master/samples/cors_sample.yaml) .
```
Comment (menakaj) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/enabling-cors.md:5]:
```suggestion
If you are following the dev-first approach ([Deploy API via CLI tool]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect/#via-the-cli-tool-apictl))You can add CrossOrigin Resource Sharing ( **CORS** ) configurations for each API (at API level) using the OpenAPI vendor extension **x-wso2-cors** in the API definition. The following code snippet depicts the usage of the `         x-wso2-cors        ` extension. For more information, see the [detailed sample OpenAPI definition with CORS level configuration](https://github.com/wso2/product-microgateway/blob/master/samples/cors_sample.yaml) .
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/enabling-cors.md [hunk 2]
```diff
@@ -23,15 +23,27 @@ x-wso2-cors:
   accessControlAllowCredentials: true
 ```
 
-### Enable CORS configuration for endpoints
+If you are following the [Deploy API via APIM](https://apim.docs.wso2.com/en/4.0.0/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect/#via-api-manager) approach, you can add **CORS** configurations for each API using the [APIM](https://apim.docs.wso2.com/en/4.0.0/design/advanced-topics/enabling-cors-for-apis/#EnablingCORSPerAPI)
```
Comment (menakaj) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/enabling-cors.md:26]:
```suggestion
If you are following the [Deploy API via APIM]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect/#via-api-manager) approach, you can add **CORS** configurations for each API using the [APIM]({{base_path}}/design/advanced-topics/enabling-cors-for-apis/#EnablingCORSPerAPI)
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/enabling-cors.md [hunk 3]
```diff
@@ -23,15 +23,27 @@ x-wso2-cors:
...
 
-You can enable CORs for /authorize, /revoke, /token, /apikey, /userinfo and /health endpoints by enabling the following configuration in `toolkit-config.toml` located in `<MICRO-GW_TOOLKIT_HOME>/conf/.`
+### Enable CORS configurations globally
+
+You can enable CORS for Choreo Connect by configuring in the `config.toml` file, which is located in the `CHOREO-CONNECT_HOME/docker-compose/choreo-connect/conf/` directory.
+
+Follow the instructions below to enable CORS globally. Once this is enabled, it will apply this configurations through all endpoints and APIs deployed in Choreo Connect.
+
+1. Open the `CHOREO-CONNECT_HOME/docker-compose/choreo-connect/conf/config.toml` file.
+2. Locate the following configuration set and make the `enabled` attribute to `true` with the required CORS attributes there.
```
Comment (VirajSalaka) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/enabling-cors.md:35]: Let's put a note saying that if cors for a certain API is disabled from API Level Configurations, the global Configurations will apply.

Comment (VajiraPrabuddhaka) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/enabling-cors.md:35]: Added a note

**Review by menakaj — APPROVED** [review]

**Review by Mariangela — APPROVED** [review]

**Pull request merged**